validators/terravaloper1k3202kcypl5csdtyvr3qjmxzzjnyn84ghj2r7y/profile.json

```diff
@@ -0,0 +1,6 @@
+{
+  "contact": {
+    "email": "admin@terrarium.community",
+    "twitter": "@ClanMudhorn_",
```

Comment:
@hanahorangi Please remove the trailing comma to adhere to JSON syntax and then we can merge your profile.  Thanks!